microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -77,8 +77,6 @@ To sign up for the **Ad Selection API** origin trial for a domain:
 
```

Comment:
could add detail at line 76
`* In the **Subdomain Support** section, select the **Yes** or **No** option button.`

it's possible to add a lot of detail here, per notes that were at bottom of old portal, rewritten at: step 5 & 6 in section:
https://learn.microsoft.com/en-us/microsoft-edge/origin-trials/#registering-for-an-origin-trial

Reply:
here's the markdown to possibly incorp, from
GitHub\edge-developer\microsoft-edge\origin-trials\index.md --
```
1. In the **Domain** text box, enter a domain URL (such as `https://example.com`) or a subdomain URL (such as `https://beta.example.com`).

   You can configure the origin trial token to support any of the following:
   * A single domain, such as `https://example.com`, without supporting its subdomains.
   * A single subdomain, such as `https://beta.example.com`.
   * A domain, such as `https://example.com`, and its subdomains, such as `https://beta.example.com`.

   Trailing paths and query parameters aren't supported.  If a URI is entered that has a trailing path or a query parameter, such as `https://example.com/path/new-feature`, the root domain (such as `https://example.com`) or subdomain will be registered, ignoring any trailing path or query parameters.

1. Set the **Enable token for subdomains?** toggle.

   * To enable using the origin trial at a domain (such as `https://example.com`) without supporting its subdomains (such as `https://beta.example.com`), make sure the **Enable token for subdomains?** toggle is turned off.

   * To enable using the origin trial only at a specific subdomain (such as `https://beta.example.com`), make sure the **Enable token for subdomains?** toggle is turned off.

   * To enable using the origin trial at a domain (such as `https://example.com`) and its subdomains (such as `https://beta.example.com`), turn on the **Enable token for subdomains?** toggle.
```